ui/components/multichain/import-tokens-modal/index.scss

```diff
@@ -11,17 +37,66 @@
     }
   }
 
+  &__modal-content {
+    padding-top: 200px;
```

Comment:
I think we can remove both of these padding values and trust the modal to be where it's supposed to.

Reply:
In the designs the Modal is supposed to show at the center of the screen; the default behavior makes it land a bit on the top of the screen; 
I could double check again with our designer if we could stick with the default landing behavior of the modals